code/addons/highlight/package.json

```diff
@@ -23,9 +23,22 @@
   },
   "license": "MIT",
   "author": "winkerVSbecks",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./preview": {
+      "require": "./dist/preview.js",
+      "import": "./dist/preview.mjs",
+      "types": "./dist/preview.d.ts"
```

Comment:
I guess these should point to `/dist/highlight`, right?  

A good way to check this is to run `yarn prep` from this folder, and then see what's getting put into `/dist`.  It should only be the files that you specify in the `entries` setting.

Reply:
Should ./preview point to dist/highlight or do I change ./preview to ./highlight?

Reply:
I think we want preview imports to point to highlight (since that's what the `/preview.js` file was doing before), but @ndelangen could confirm.

Reply:
Done. I made the change